src-vue/src/plugins/store.ts

```diff
@@ -77,17 +78,36 @@ export const store = createStore({
                 return;
             }
 
-            // Show an error message if Origin is not running.
-            if (!state.origin_is_running) {
-                ElNotification({
-                    title: 'Origin is not running',
-                    message: "Northstar cannot launch while you're not authenticated with Origin.",
-                    type: 'warning',
-                    position: 'bottom-right'
-                });
-            }
+            // Game is ready to play
+            if (state.northstar_state === NorthstarState.READY_TO_PLAY) {
+                // Show an error message if Origin is not running.
+                if (!state.origin_is_running) {
```

Comment:
Origin must be running for Steam installations as well, right?

Reply:
What about EA Desktop installations, by the way?

Reply:
> Origin must be running for Steam installations as well, right?

Yup, if you launch via `NorthstarLauncher.exe` directly (which is what the backend currently does), then yes.

> What about EA Desktop installations, by the way?

I haven't tested them myself yet, for now this is handled on the back-end where it simply errors out if the install is EA Desktop: https://github.com/GeckoEidechse/FlightCore/blob/93cec9e9cf8d1f67c38e4194b6c912aff06a3f22/src-tauri/src/lib.rs#L251-L253

In the future, I would move all the checks (including the one to see if Origin is running) to the backend cause it allows me to handle all the logic there to launch Northstar differently depending on the platform (Origin/Steam/EA-Desktop on Windows/Linux) without having to jump between frontend and backend ^^